ci/taos/plugins-good/pr-format-prohibited-words.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+##
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+##
+# @file     pr-format-prohibited-words.sh
+# @brief    Check if there are prohibited words in the text files
+#
+# It to check a prohibited word if there are unnecessary words in the source codes.
+#
+# @see      https://github.com/nnsuite/TAOS-CI
+# @author   Geunsik Lim <geunsik.lim@samsung.com>
+#
+
+# @brief [MODULE] TAOS/pr-format-prohibited-words
+function pr-format-prohibited-words(){
+    echo -e "########################################################################################"
+    echo -e "[MODULE] TAOS/pr-format-prohibited-words: Check if there are prohibited words in the text files."
+    pwd
+
+    # Check if a server administrator install required commands.
+    check_dependency git
+    check_dependency grep
+    check_dependency wc
+
+    # Read file names that a contributor modified(e.g., added, moved, deleted, and updated) from a last commit.
+    FILELIST=`git show --pretty="format:" --name-only --diff-filter=AMRC`
+
+    # Inspect all files that contributor modifed.
+    target_files=""
+    for i in ${FILELIST}; do
+        # Default value of check_result is "skip".
+        check_result="skip"
+
+        # Skip obsolete folder
+        if [[ $i =~ ^obsolete/.* ]]; then
+            continue
+        fi
+        # Skip external folder
+        if [[ $i =~ ^external/.* ]]; then
+            continue
+        fi
+        # Handle only text files among files in one commit.
+        echo -e "[DEBUG] file name is ( $i )."
+        if [[ `file $i | grep "ASCII text" | wc -l` -gt 0 ]]; then
+            case $target_files in
```

Comment:
I think $i is proper than $target_files.

Reply:
Nice catch. Fixed.